index.js

```diff
@@ -309,9 +321,50 @@ export class Placement {
           clearInterval(viewport_detection);
         }
       }, 100, placement.target);
+
+      let view_time_counter = setInterval((element) => {
+        if (placement.view_time_sent) {
+          clearInterval(view_time_counter);
+        } else if (placement.inViewport(element)) {
+          // Increment the ad's time in view counter
+          placement.view_time += VIEW_TIME_INTERVAL;
+
+          if (placement.view_time >= VIEW_TIME_MAX) {
+            clearInterval(view_time_counter);
+          }
+        }
+      }, VIEW_TIME_INTERVAL * 1000, placement.target);
+
+      document.addEventListener("visibilitychange", () => {
+        // Check if the tab has gone out of focus or the browser/app is minimized
```

Comment:
Does this also trigger on closing of the tab?

Reply:
Yes. It is supposed to and it did successfully in Chrome as far as I could test in the debugger.

Reply:
I wouldn't mind running this whole thing through an IE11 test as well.

Reply:
We should note that in the comment then, since I imagine that's the most important time this will be fired.

Reply:
Done